graylog2-web-interface/src/components/common/Wizard.jsx

```diff
@@ -114,30 +147,40 @@ class Wizard extends React.Component {
             </Button>
           </ButtonToolbar>
         </div>
-        <Nav bsStyle="pills" activeKey={this.state.selectedStep} onSelect={this._wizardChanged}>
-          {this.props.steps.map((navItem) => {
-            return (<NavItem key={navItem.key} eventKey={navItem.key} disabled={navItem.disabled}>{navItem.title}</NavItem>);
+        <Nav bsStyle="pills"
+             className={navigationClassName}
+             activeKey={selectedStep}
+             onSelect={this._wizardChanged}
+             justified={justified}>
+          {steps.map((navItem) => {
+            return (
+              <NavItem key={navItem.key} eventKey={navItem.key} disabled={navItem.disabled}>{navItem.title}</NavItem>);
           })}
         </Nav>
       </Col>
     );
   };
 
+  _getSelectedStep() {
+    const { activeStep } = this.props;
+    const { selectedStep } = this.state;
+    return lodash.defaultTo(activeStep, selectedStep);
```

Comment:
To avoid loading in lodash, could this just be `return activeStep || selectedStep;` ?

Reply:
The only problem I see is if `activeStep` is a number, and for some reason needs to be 0. We can always restrict the key to only be a string, but for now I thought it was safer going with `defaultTo()`.